src/ontology/mondo-edit.obo

```diff
@@ -208346,8 +208345,9 @@ property_value: confidence "0.18928571428571428" xsd:double
 
 [Term]
 id: MONDO:0010764
-name: deafness, Y-linked 1
+name: hearing loss, Y-linked 1
 synonym: "deafness, Y-linked 1" EXACT [MONDO:Lexical, OMIM:400043]
+synonym: "deafness, Y-linked 1" NARROW []
```

Comment:
the old labels/now synonyms should have the OMIM ID as the dbxref. I started manually fixing these, I'm not sure if it is possible to automate this?
If not, I'll fix them manually.

Reply:
Yeah, that was fun. I think I did it.